src/components/pages/Navigation.tsx

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
+import { BrowserRouter, Route, Switch, Link } from 'react-router-dom';
+import NameForm from './StatedNameForm';
+import GraphForm from './StatedGraphForm';
 
 const Navigation = () => {
   return (
     <div>
-      <NavLink to="/">Home </NavLink>
-      <NavLink to="/graph">graph</NavLink>
+      <h2>
+        <BrowserRouter>
+          <h2>Welcome to cashburn, app for budget management</h2>
+          <li>
```

Comment:
You should use <li> in <ol> or <ul> tag to group them

Reply:
Its not doing anything. At least I dont see any change

Reply:
ol = ordered list
ul = unordered list

It is styling hint mostly - I recommended it only to group entries better.

Reply:
I searched the net out of curiosity and agree with the opinions that it depends on the browser and some may be a bit distracted. The browser is also such a smart tool that it probably fixes the incomplete code for the developer ;)

Reply:
Ok, I think it is clear now. Funny cos u dont see the effect, not like the browser